### ISSUE TITLE
Feature Request Options on Right Click Tab 

### ISSUE THREAD

**Issue by Bamajon1974** [rogerskeie/autoMutePlus#4]
I have a feature request for auto mute plus: can an option be implemented to show the right hand context menu on either a web page right click or tab right click or both? 

Thanks in advance

**Comment by rogerskeie** [comment]
Hi Bajamon,

I'm not sure what you mean, because there is already an "Auto Mute Plus" menu item in the context menu when you right click on a webpage, containing "Add domain to whitelist", "Add URL to whitelist" etc., 6 menu items in total.

Do you mean an option to have these menu items in the root of the context menu instead of under the Auto Mute Plus submenu?

**Comment by Bamajon1974** [comment]
No, in this context, what I mean is that there is an option to, when you
right-click on a tab in the tab bar of Firefox, you can access the Auto
Mute Plus options rather than right click anywhere else on a page for the
options. That way, tab related functions can remain separate of other
search related functions on the context menu and reducing clutter. An
option to add the AutoMute Plus options to the standard webpage and/or tab
context menus would be nice. Please see the attached picture.

Does that make sense? Apparently Firefox lets you add items to the right
click tab menu but Chrome does not.

[image: image.png]

**Comment by rogerskeie** [comment]
I can't see the picture you attached (it just says [image: image.png]), but I think I understand.

You want a couple of check boxes in the plugin options that say something like "Add to tab context menu" and "Add to page context menu", right?

I'll add it to the list of features to implement in the next version.

**Comment by Bamajon1974** [comment]
Correct! That way tab related functions right-click extensions can be
accessed in the tab menu to keep the regular page related right click
functions separate and help reducing context menu bloat.

Let me know when you release a new version and I will test it for you.

Thanks, Jon
